examples/containers/ModalPicker.tsx

```diff
@@ -23,7 +23,7 @@ export default function ModalPicker() {
 
     const { year, month } = newValue
     const newDayArray = getDayArray(Number(year), Number(month))
-    const newDay = newDayArray.includes(newValue.day) ? newValue.day : newDayArray[newDayArray.length - 1]
+    const newDay = newDayArray.includes(newValue.day as string) ? newValue.day : newDayArray[newDayArray.length - 1]
```

Comment:
Had to add this coercion to the test. This could possibly be avoided through type inference or generics, instead of just a union as I added: `string | number`.

Interested in your thoughts @adcentury